src/reducers/data.js

```diff
@@ -5,36 +5,40 @@ const app = (state = {cards:{}, stats:{}, categories:[]}, action) => {
   let json, value;
   switch (action.type) {
     case UPDATE_CARDS:
-      return {
-        ...state,
-        cards: {...state.cards, [action.hint]: action.cards}
-      }
+      return Object.assign({}, state, {[action.hint]: action.cards});
```

Comment:
I got some errors running on this branch after clearing all the storages.
```
Uncaught (in promise) TypeError: Cannot read property 'length' of undefined
    at pickCardFromCategories (data.js:125)
```
It looks like we should be rewritten like this instead:
```
return Object.assign({}, state, {
  cards: Object.assign({}, state.cards, {
    [action.hint]: action.cards
  })}
);
```